docs/generated/openapi.yaml

```diff
@@ -3129,9 +3153,26 @@ components:
             $ref: '#/components/schemas/ResourceRule'
         fromRules:
           type: array
-          description: a set of rules for each inbound of this proxy
+          description: >-
+            a set of rules for each inbound of this proxy. The field is not set
+            when 'interpretFromEntriesAsRules' on PolicyDescriptor is set to
+            true.
           items:
             $ref: '#/components/schemas/FromRule'
+        inboundRules:
+          type: array
+          description: >-
+            a set of rules for each inbound of the proxy. When policy descriptor
+            has 'interpretFromEntriesAsRules' set to true, this field replaces
+            'fromRules'.
+          items:
```

Comment:
Where will the actual sub rules be?

Reply:
Sorry, not sure I follow, what're the sub rules? 

The example of the following schema is:

```yaml
inboundRules:
  - inbound:
      port: 8080
      name: http-api
      tags: {}
    rules:
      - conf: $conf1
        origin: [...]
  - inbound:
      port: 9091
      name: tcp-server
      tags: {}
    rules:
      - conf: $conf2
        origin: [...]
```